README.md

```diff
@@ -581,6 +583,27 @@ Class2       0            10
 
 ```	
 
+### Plot
+`plot` method is added in `version 3.0` in order to plot a confusion matrix by use of matplotlib or seaborn.
```

Comment:
Please change it to:
`plot` method is added in `version 3.0` in order to plot a confusion matrix using Matplotlib or Seaborn.